FutureNorthants/northants-website
# Fix scope issue and sort news desc

The `getNewsFeed()` method called getSizedImageURL() on the article node, but the method was protected in the trait. Updated to public so it can be called from a different scope. 

Also updated the sort order so new items will appear at the top of the feed instead of the end. 

## To Test

Use the below query and ensure that it returns data, and that the entries are listed in date descending order.

```graphql
{
  news_feed {
    __typename
    title
    description
    language
    updated
    entries {
      title
      id
      date
      content
      link
    }
  }
}
```
